source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_env.py

```diff
@@ -251,6 +251,8 @@ def reset(self, seed: int | None = None, options: dict[str, Any] | None = None)
         indices = torch.arange(self.num_envs, dtype=torch.int64, device=self.device)
         self._reset_idx(indices)
 
+        # update articulation kinematics
```

Comment:
I think this should be called after the event manager is called to reset transforms (so all the other managers also get this change)

> https://github.com/isaac-sim/IsaacLab/blob/5ab1f0bf57540fbfb6fddb562ca99612940c33a0/source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_env.py#L363